e2e-tests/popup/account/remove-imported-account.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ACCOUNT_NAMES,
+  IMPORTED_PEM_ACCOUNT,
+  secretKeyPathForPEM
+} from '../../constants';
+import { popup, popupExpect } from '../../fixtures';
+
+popup.describe('Popup UI: remove imported account', () => {
+  popup(
+    'should remove imported account',
+    async ({ unlockVault, context, popupPage }) => {
+      await unlockVault();
+
+      await popupPage.getByTestId('menu-open-icon').click();
+
+      const [importAccountPage] = await Promise.all([
+        context.waitForEvent('page'),
+        popupPage.getByText('Import account').click()
+      ]);
+
+      const fileChooserPromise = importAccountPage.waitForEvent('filechooser');
+
+      await popupExpect(
+        importAccountPage.getByRole('heading', {
+          name: 'Import account from secret key file'
+        })
+      ).toBeVisible();
+
+      await importAccountPage
+        .getByRole('button', { name: 'Upload your file' })
+        .click();
+
+      const fileChooser = await fileChooserPromise;
+      await fileChooser.setFiles(secretKeyPathForPEM);
+
+      await importAccountPage
+        .getByPlaceholder('Account name', { exact: true })
+        .fill(ACCOUNT_NAMES.importedPemAccountName);
+
+      await importAccountPage.getByRole('button', { name: 'Import' }).click();
+
+      await popupExpect(
+        importAccountPage.getByRole('heading', {
+          name: 'Your account was successfully imported'
+        })
+      ).toBeVisible();
+
+      await importAccountPage.getByRole('button', { name: 'Done' }).click();
+
+      await popupPage.getByTestId('connection-status-modal').click();
+
+      await popupExpect(
+        popupPage.getByText(ACCOUNT_NAMES.importedPemAccountName)
+      ).toBeVisible();
+      await popupExpect(
+        popupPage.getByText(IMPORTED_PEM_ACCOUNT.truncatedPublicKey)
+      ).toBeVisible();
+      await popupExpect(
+        popupPage.getByTestId('import-account-icon')
+      ).toBeVisible();
+
+      await popupPage
+        .locator('div:nth-child(3) > div > div > div:nth-child(3) > .sc-lbVpMG')
```

Comment:
create a test id. I can't understand what button we are clicking.
If in the future we change some structure, this selector will break